psasim/test/Makefile

```diff
@@ -4,19 +4,25 @@ LIB := -L../src -lpsaff
 TEST_BIN = 	psa_client \
 			psa_partition
 
+GENERATED_H_FILES =	./psa_manifest/manifest.h \
+					./psa_manifest/pid.h \
+					./psa_manifest/sid.h
+
 .PHONY: all clean
 
 all: $(TEST_BIN)
 
-psa_client:
+psa_client: $(GENERATED_H_FILES)
```

Comment:
If we're going to add explicit dependencies when there's the compilation rules, can we include `client.c` in this list please :)

Reply:
I added those explicit dependencies because those need to be auto-generated. I didn't add already existing source files because I assumed that this was not necessary.
Following you suggestion then I would also need to check for `server.c` when building `psa_partition`, isn't it?